admin/src/components/Action/ActionDateTimePicker/ActionDateTimePicker.tsx

```diff
@@ -22,51 +23,63 @@ const ActionDateTimePicker = ({ executeAt, mode, isCreating, isEditing, onChange
 	const { isLoading, data, isRefetching } = getSettings();
 
 	useEffect(() => {
-		if (!isLoading && !isRefetching) {
+		if (! isLoading && ! isRefetching) {
```

Comment:
Ideally we migrate this plugin to be on our own eslint rules. Because then this would fail.
I'm not a big fan of this way of writing logical NOT operators